modules/apps/transfer/keeper/keeper.go

```diff
@@ -30,10 +30,12 @@ type Keeper struct {
 	cdc            codec.BinaryCodec
 	legacySubspace types.ParamSubspace
 
-	ics4Wrapper   porttypes.ICS4Wrapper
-	channelKeeper types.ChannelKeeper
-	AuthKeeper    types.AccountKeeper
-	BankKeeper    types.BankKeeper
+	ics4Wrapper     porttypes.ICS4Wrapper
+	channelKeeper   types.ChannelKeeper
+	channelKeeperV2 types.ChannelKeeperV2
+	clientKeeperV2  types.ClientKeeperV2
```

Comment:
I could remove this if desired

Reply:
It would be nice to have less stuff to put in here, but what would that mean for the code?